spec/services/proofing/aamva/verification_client_spec.rb

```diff
@@ -37,45 +39,126 @@
 
       expect(verification_stub).to have_been_requested
     end
+  end
 
-    context 'when verification is successful' do
-      it 'should return a successful response' do
-        auth_client = instance_double(Proofing::Aamva::AuthenticationClient)
-        allow(auth_client).to receive(:fetch_token).and_return('ThisIsTheToken')
-        allow(Proofing::Aamva::AuthenticationClient).to receive(:new).and_return(auth_client)
-        stub_request(:post, AamvaFixtures.example_config.verification_url).
-          to_return(body: AamvaFixtures.verification_response, status: 200)
-
-        response = verification_client.send_verification_request(
-          applicant: applicant,
-          session_id: '1234-abcd-efgh',
-        )
+  describe '#send_verification_request' do
```

Comment:
This `describe` block looks like it is describing the same method as the block above

Reply:
It is, but a different aspect of the method. The setup code was different enough to warrant splitting the describe block.